VirtualReality/qSlicerVirtualRealityModuleWidget.cxx

```diff
@@ -145,3 +157,33 @@ void qSlicerVirtualRealityModuleWidget::setBackLights(bool active)
     vrViewNode->SetBackLights(active);
   }
 }
+
+//-----------------------------------------------------------------------------
+void qSlicerVirtualRealityModuleWidget::setReferenceViewNode(vtkMRMLNode* node)
```

Comment:
Why do we need this function? Just for convenience?

Reply:
When user selects the reference view node on the GUI then this slot is called to update the view node.

Reply:
Aah it's a slot! Gotcha, thanks!